charts/spire/values.yaml

```diff
@@ -36,54 +40,88 @@ global:
+  ## @param spire-agent.nameOverride Overrides the name of Spire agent pods
   nameOverride: agent
 
+## @section Upstream Spire agent and CSI driver configuration
+## Parameter values enabling upstream spire agent and CSI driver
+##
 upstream:
-  # -- enable upstream agent and driver for use with nested spire.
+  ## @param upstream.enabled Enable upstream agent and driver for use with nested spire
   enabled: false
 
+## @section Upstream Spire agent parameters
+## Parameter values for upstream Spire agent
+##
 upstream-spire-agent:
-  # @ignored
+  ## @param upstream-spire-agent.upstream Flag for enabling upstream Spire agent
   upstream: true
+  ## @param upstream-spire-agent.nameOverride Name override for upstream Spire agent
   nameOverride: agent-upstream
+  ## @param upstream-spire-agent.bundleConfigMap The configmap name for upstream Spire agent bundle
   bundleConfigMap: spire-bundle-upstream
-
+  ## @param upstream-spire-agent.socketPath Socket path where Spire agent socket is mounted
   socketPath: /run/spire/agent-sockets-upstream/spire-agent.sock
   serviceAccount:
+    ## @param upstream-spire-agent.serviceAccount.name Service account name for upstream Spire agent
     name: spire-agent-upstream
   healthChecks:
+    ## @param upstream-spire-agent.healthChecks.port Health check port number for upstream Spire agent
     port: 9981
   telemetry:
     prometheus:
+      ## @param upstream-spire-agent.telemetry.prometheus.port The port where prometheus metrics are available
       port: 9989
 
+## @section Spire CSI Driver parameters
+## Parameter values for spire-csi-driver
+##
 spiffe-csi-driver:
-  # -- Enables deployment of CSI driver
+  ## @param spiffe-csi-driver.enabled Flag to enable spiffe-csi-driver for the cluster
   enabled: true
 
+## @section Upstream Spire CSI Driver parameters
+## Parameter values for upstream spire-csi-driver
+##
 upstream-spiffe-csi-driver:
+  ## @param upstream-spiffe-csi-driver.pluginName The plugin name for configuring upstream Spiffe CSI driver
   pluginName: upstream.csi.spiffe.io
+  ## @param upstream-spiffe-csi-driver.agentSocketPath The socket path where Spiffe CSI driver mounts agent socket
   agentSocketPath: /run/spire/agent-sockets-upstream/spire-agent.sock
   healthChecks:
+    ## @param upstream-spiffe-csi-driver.healthChecks.port The port where Spiffe CSI driver health checks are exposed
     port: 9810
 
+## @section SPIFFE oidc discovery provider parameters
+## Parameter values for spiffe-oidc-discovery-provider
+##
 spiffe-oidc-discovery-provider:
-  # -- Enables deployment of OIDC discovery provider
+  ## @param spiffe-oidc-discovery-provider.enabled Flag to enable spiffe-oidc-discovery-provider for the cluster
   enabled: false
 
+## @section Tornjak frontend parameters
+## Parameter values for Tornjak frontend
+##
 tornjak-frontend:
-  # -- Enables deployment of Tornjak frontend/UI (Not for production)
+  ## @param tornjak-frontend.enabled Flag to enable Tronjak frontend
```

Comment:
Do we stil want to keep the not for production marker here? cc @mrsabath

Reply:
I think we probably want to keep it that way until we have oidc support in it?

Reply:
Have changed the param description to reflect this (not for production).